hr_simplify/tests/test_hr_simplify.py

```diff
@@ -0,0 +1,153 @@
+
+
+class TestHrSimplify(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrSimplify, cls).setUpClass()
+
+        # -- models
+        cls.Contract = cls.env["hr.contract"]
+        cls.Department = cls.env["hr.department"]
+        cls.Employee = cls.env["hr.employee"]
+        cls.Job = cls.env["hr.job"]
+
+        # -- departments
+        cls.dept_marketing = cls.Department.create({"name": "Marketing"})
+        cls.dept_RandD = cls.Department.create({"name": "Research and Development"})
+
+        # -- jobs
+        # -- marketing
+        cls.job_mkt_director = cls.Job.create({"name": "Marketing Director"})
+
+        # -- R&D
+        cls.job_rnd_manager = cls.Job.create(
+            {
+                "name": "Research and Development Manager",
+                "no_of_recruitment": 2,
+            }
+        )
+        cls.job_rnd_technician = cls.Job.create(
+            {
+                "name": "Research and Development Technician",
+                "no_of_recruitment": 4,
+            }
+        )
+
+        # -- employees
+        cls.emp_adam = cls.Employee.create(
+            {
+                "name": "Adam",
+                "gender": "male",
+                "department_id": cls.dept_marketing.id,
+                "job_id": cls.job_mkt_director.id,
+                "active": True,
+            }
+        )
+
+        cls.emp_kal = cls.Employee.create(
+            {
+                "name": "Kal",
+                "gender": "female",
+                "department_id": cls.dept_RandD.id,
+                "job_id": cls.job_rnd_manager.id,
+                "active": True,
+            }
+        )
+
+        cls.emp_tim = cls.Employee.create(
+            {
+                "name": "Tim",
+                "gender": "male",
+                "department_id": cls.dept_RandD.id,
+                "job_id": cls.job_rnd_technician.id,
+                "active": True,
+            }
+        )
+
+        cls.emp_ed = cls.Employee.create(
+            {
+                "name": "Ed",
+                "gender": "male",
+                "department_id": cls.dept_RandD.id,
+                "job_id": cls.job_rnd_technician.id,
+                "active": True,
+            }
+        )
+
+        # -- contract data
+        cls.contract_data = {
+            "state": "open",
+            "kanban_state": "normal",
+            "wage": 1,
+            "date_start": False,
+            "date_end": False,
+        }
+
+    def create_contract(
+        self, employee, job, start_date=None, end_date=None, name="Contract#1"
+    ):
+        self.contract_data["name"] = name
+        self.contract_data["employee_id"] = employee.id
+        self.contract_data["job_id"] = job.id
+        self.contract_data["date_start"] = (
+            datetime.now().date() if start_date is None else start_date
+        )
+        self.contract_data["date_end"] = (
+            (datetime.now() + timedelta(days=90)).date()
```

Comment:
same here. This could be more legibly written as:
fields.Date.today() + timedelta(days=90)